test/e2e/storage/testsuites/provisioning.go

```diff
@@ -425,6 +426,88 @@ func (p *provisioningTestSuite) DefineTests(driver storageframework.TestDriver,
 		l.testCase.TestDynamicProvisioning()
 	})
 
+	ginkgo.It("should provision correct filesystem size when restoring snapshot to larger size pvc [Feature:VolumeSnapshotDataSource]", func() {
+		if pattern.VolMode == "Block" {
+			e2eskipper.Skipf("Test is not valid for Block volume mode - skipping")
+		}
+
+		if dInfo.Capabilities[storageframework.CapFilesystemResizeNotSupported] {
```

Comment:
I think this could be in the PD CSI Driver linux capabilities but not in the Windows one as this command `cmd := fmt.Sprintf("stat -f -c %%s %v", mountPath)` won't work in Windows.

Reply:
It looks like this provisioning test will run by default in Windows because of the anti-capacity (i.e. because the test is opt-out), it'll take some effort to make it run in Windows, I'd add a tracking issue and a skip with:

```
  if framework.NodeOSDistroIs("windows") {
    e2eskipper.Skipf(...)
  }
```

Reply:
Added a skip with a comment and link to https://github.com/kubernetes/kubernetes/issues/113359